src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java

```diff
@@ -81,6 +81,11 @@ private JavaParser() {
      */
     public static DetailAST parse(FileContents contents)
             throws CheckstyleException {
+        final String filename = contents.getFileName();
+        if (filename.contains("indentation/indentation/Input")) {
+            throw new CheckstyleException("Exception!");
+        }
```

Comment:
@romani one run should be enough, if no exception is thrown, then exclusions work